src/Nodes/TwinNode.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as vscode from "vscode";
+import { TwinItem } from "../Model/TwinItem";
+import { DistributedTracingSettingNode } from "./DistributedTracingSettingNode";
+import { INode } from "./INode";
+import iothub = require("azure-iothub");
+import { Constants, DeviceTwinPropertyType } from "../constants";
+import { TelemetryClient } from "../telemetryClient";
+import { Utility } from "../utility";
+import { DeviceNode } from "./DeviceNode";
+export class TwinNode implements INode {
+    private readonly DISTRIBUTED_TRACING_ENABLED_PROPERTY: string = "Enable Distributed Tracing: ";
+    private readonly DISTRIBUTED_TRACING_SAMPLING_RATE: string = "Sampling Rate: ";
+
+    constructor(
+        private twinItem: TwinItem,
+        public deviceNode: DeviceNode) {
+    }
+
+    public getTreeItem(): vscode.TreeItem {
+        return this.twinItem;
+    }
+
+    public async getChildren(context: vscode.ExtensionContext, iotHubConnectionString: string): Promise<INode[]> {
+        let registry = iothub.Registry.fromConnectionString(iotHubConnectionString);
+
+        TelemetryClient.sendEvent(Constants.IoTHubAILoadDistributedTracingSettingStartEvent, null, iotHubConnectionString);
+        const items: INode[] = [];
+
+        try {
+            let twin = await Utility.getTwin(registry, this.deviceNode.deviceId);
+            let samplingRate = null;
+            let enabled = null;
+            if (this.twinItem.type === DeviceTwinPropertyType.Desired) {
+                if (twin.properties.desired[Constants.DISTRIBUTED_TWIN_NAME]) {
+                    samplingRate = Utility.parseDesiredSamplingRate(twin);
+                    enabled = Utility.parseDesiredSamplingMode(twin);
+
+                    items.push(new DistributedTracingSettingNode(this.DISTRIBUTED_TRACING_ENABLED_PROPERTY + (enabled ? "Enabled" : "Disabled"), this, "desired-mode-property", this.deviceNode));
+                    items.push(new DistributedTracingSettingNode(this.DISTRIBUTED_TRACING_SAMPLING_RATE +
+                        (samplingRate !== undefined ? samplingRate + "(%)" : "Not Set"), this, "desired-sampling-rate-property", this.deviceNode));
```

Comment:
Will ` %` be better? And better put ` %` and "Not Set" into contant

Reply:
Also 'Disabled' and 'Enabled'

Reply:
fixed